app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md

```diff
@@ -6,8 +6,9 @@ description: |-
   Find out about the assessment only route to qualified teacher status (QTS) for unqualified teachers who have worked in a classroom.
 image: false
 backlink: /
-promo_content:
-    - content/train-to-be-a-teacher/promos/adviser-promo-assessment-only
+cta_adviser:
```

Comment:
I can't see this on the front end? Check with Martyn as to how this needs to work!

Reply:
@emmaachesongray in addition to defining the adviser promo here in the frontmatter, you also need to call the adviser promo in the markdown (at the bottom of this file) with `$adviser$`

Reply:
Furthermore, this file is generated from a template, and the template will need to be updated too: `lib/tasks/support/assessment-only-route-to-qts.md.erb`